packages/ember-glimmer/lib/helpers/readonly.js

```diff
@@ -5,6 +5,101 @@
 import { UPDATE } from '../utils/references';
 import { unMut } from './mut';
 
+
+/**
+  The `readonly` helper let's you specify that when you updated a value passed in
+  to a child component, it does not update in the parent component.
+  
+  To specify that a parameter is read-only, when invoking the child `Component`:
+  
+    ```javascript
+  // my-parent.js
+  export default Component.extend({
+    totalClicks: 3
+  });
+  ```
+  
+  ```handlebars
+  {{log totalClicks}} // -> 3
+  {{my-child childClickCount=(readonly totalClicks)}}
+  ```
+  
+  Now, when you update `childClickCount`:
+  
+  ```javascript
+  // my-child.js
+  export default Component.extend({
+    click() {
+      this.incrementProperty('childClickCount');
+    }
+  });
+  ```
+  
+  The value updates in the child component, but not the parent component:
```

Comment:
I think the current text is good, or at the least if something like the suggested change was used it should be specific to the property names used in the example.